test/index.js

```diff
@@ -101,6 +101,15 @@ describe("depcheck", function () {
     });
   });
 
+  it("should report bad javascript", function testBadJS(done) {
+    var absolutePath = path.resolve("test/fake_modules/bad_js");
+
+    depcheck(absolutePath, {  }, function checked(unused) {
+      assert.notEqual(unused.invalidFiles, {});
```

Comment:
Alright, cleaned it up in https://github.com/woodb/depcheck-es6/commit/1ac996178046b5db860f006cf09844bfb4ede2aa :+1: